lib/services/northBound/northboundServer.js

```diff
@@ -117,7 +117,6 @@ exports.setNotificationHandler = intoTrans(context, contextServer.setNotificatio
 exports.setConfigurationHandler = intoTrans(context, groupProvisioning.setConfigurationHandler);
 exports.setRemoveConfigurationHandler = intoTrans(context, groupProvisioning.setRemoveConfigurationHandler);
 exports.setProvisioningHandler = intoTrans(context, deviceProvisioning.setProvisioningHandler);
-exports.setUpdatingHandler = intoTrans(context, deviceUpdating.setUpdatingHandler);
```

Comment:
This change is not included in PR https://github.com/telefonicaid/iotagent-node-lib/pull/954

Could you elaborate why is included here?